plugins/scanners/askalono/src/main/kotlin/Askalono.kt

```diff
@@ -107,7 +105,7 @@ class Askalono(
     }
 
     override fun createSummary(result: String, startTime: Instant, endTime: Instant): ScanSummary {
-        val results = result.byteInputStream().use { JSON.decodeToSequence<AskalonoResult>(it) }
+        val results = result.byteInputStream().use { Json.decodeToSequence<AskalonoResult>(it) }
```

Comment:
If one wants to generally ignore unkown keys, for fault tolerance, why shouldn't one set it in a single place?

This would prevent bugs related to missing `@JsonIgnoreUnknownKeys`, and it would be less verbose.

Reply:
As written in the commit message, I find it clearer if this is explicitly stated next to the class. Also, this way it's basically format-agnostic: Using the same data classes for YAML- instead of JSON-deserialization should also ignore unknown properties, without the need to also configure the YAML serial format accordingly (even if the annotation is called "*Json*IgnoreUnknownKeys").

Reply:
> I find it clearer

Would this rationale then apply regardless of this?

> If the only customization is to ignore unknown keys